tasks/keylime-config.yml

```
@@ -0,0 +1,89 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Make sure verifier snippet directory exists
+  ansible.builtin.file:
+    path: "{{ __keylime_server_verifier_snippet_dir }}"
+    state: directory
+    mode: '0500'
```

```suggestion
    mode: "0500"
```
double quotes are preferred, except inside a `"{{ .... }}"` Jinja expression